genstudio-io-runtime-app/src/genstudiopem/web-src/src/Constants.ts

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
```

Comment:
I see a mix use of license header, ex. this one vs. 
```
/* 
* <license header>
*/
```
Maybe we want to align. It's a tricky one, because we assume this is a template :)

Reply:
i think we should remove all license headers and do
```
/* 
* <license header>
*/
```
instead